src/target/riscv/riscv-013.c

```diff
@@ -29,6 +29,9 @@
 #include "asm.h"
 #include "batch.h"
 
+#define DTMCONTROL					0x10
+#define DBUS						0x11
```

Comment:
DTMCONTROL is old. The register was renamed, and its value is defined in debug_defines.h as DTM_DTMCS.
Likewise DBUS is now DTM_DMI.

Reply:
ok, this is copied from riscv.c. I will fix it